digital/README.md

```diff
@@ -7,13 +7,18 @@
   </div>
 </heading>
 
+<div class="identity-section section1-padding">
```

Comment:
Rather than using a custom class you could use `py-3`: https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
Good shout!